test/server/admin/stats_request_test.cc

```diff
@@ -147,6 +156,27 @@ TEST_F(StatsRequestTest, OneStatUsedOnly) {
   EXPECT_EQ(0, iterateChunks(*makeRequest(true, StatsFormat::Text, StatsType::All)));
 }
 
+TEST_F(StatsRequestTest, OneStatHiddenExclude) {
+  store_.rootScope()->gaugeFromStatName(makeStatName("foo"),
```

Comment:
nice tests. But can you add one more non-hidden gauge and verify that it is present/absent as needed?

I realize the test pattern you are extending is just focused how StatsRequest chunks out the responses, and I'm asking you to look at the actual data.

I think in that respect you should probably leave these new tests here, but put the content-related tests into stats_handler_test.cc, where there are similar content-oriented tests.

Reply:
I added extra tests in stats_handler_test. Done